### DISCUSSION TITLE
Alternative Elsa Workflow Designer based on WPF (.NET5) and possibly (future) WebAssembly through UNO

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [elsa-workflows/elsa-core#882] [Ideas]
[Here is the link](https://github.com/miroiu/nodify/issues/6#issuecomment-818968383)

Please visit this project ASAP! 
**Miroiu Emanuel** is a very talented UI (full stack developer). 
The examples come with a highly interactive working UI engaging State machine!

Please i**mmediately give LIKE and show your support**, so Miroiu would consider supporting an alternative designer for the elsa-workflow project!

![image](https://user-images.githubusercontent.com/49812372/114769756-e5a7b680-9d6a-11eb-9725-a14fe17af80a.png)

**Comment by sfmskywalker** [comment]
That looks amazing. Thank you for sharing!
